api/src/test/java/com/microsoft/gctoolkit/time/DateTimeStampTest.java

```diff
@@ -334,134 +334,107 @@ public void timeSpanInMinutes() {
     }
 
     @Test
-    public void testCompareNullValue() {
+    void testCompareNullValue() {
         DateTimeStamp smaller = new DateTimeStamp("2018-04-04T09:10:00.586-0100");
         assertEquals(-1, smaller.compareTo(null));
     }
 
     @Test
-    public void testCompareEqualsTimeStampWithoutDateTime() {
+    void testCompareEqualsTimeStampWithoutDateTime() {
         DateTimeStamp object1 = new DateTimeStamp(123);
         DateTimeStamp object2 = new DateTimeStamp(123);
         assertEquals(0, object2.compareTo(object1));
     }
 
     @Test
-    public void testCompareGreaterTimeStampWithoutDateTime() {
+    void testCompareGreaterTimeStampWithoutDateTime() {
         DateTimeStamp smaller = new DateTimeStamp(122);
         DateTimeStamp greater = new DateTimeStamp(124);
         assertEquals(1, greater.compareTo(smaller));
     }
 
     @Test
-    public void testCompareSmallerTimeStampWithoutDateTime() {
+    void testCompareSmallerTimeStampWithoutDateTime() {
         DateTimeStamp smaller = new DateTimeStamp(123);
         DateTimeStamp greater = new DateTimeStamp(124);
         assertEquals(-1, smaller.compareTo(greater));
     }
 
-    /**
-     * This tests a case that can't happen in context of a GC log.
-     */
-    @Test
-    public void testCompareSmallerTimeStampMixed() {
-        DateTimeStamp smaller = new DateTimeStamp(123);
-        DateTimeStamp greater = new DateTimeStamp("2018-04-04T10:10:00.586-0100");
-        assertThrows(IllegalStateException.class,
-                () -> { smaller.compareTo(greater); },
-                "IllegalStateException Not Thrown");
-    }
-
     @Test
-    public void testAddingNaN() {
+    void testAddingNaN() {
         assertThrows(IllegalArgumentException.class,
                 () -> { new DateTimeStamp("2018-04-04T10:10:00.586-0100").add(Double.NaN); },
                 "IllegalAccess Not Thrown");
     }
 
     @Test
-    public void testNanWithMinusNonZero() {
+    void testNanWithMinusNonZero() {
         assertThrows(IllegalArgumentException.class,
                 () -> { new DateTimeStamp("2018-04-04T10:10:00.586-0100").minus(Double.NaN); },
                 "IllegalAccess Not Thrown");
     }
 
     @Test
-    public void testNanWithMinus() {
+    void testNanWithMinus() {
         DateTimeStamp dateTimeStamp = new DateTimeStamp("2018-04-04T10:09:59.586-0100");
         DateTimeStamp forComparing = new DateTimeStamp("2018-04-04T10:10:00.586-0100").minus(1.0d);
         assertEquals(dateTimeStamp, forComparing);
     }
 
     @Test
-    public void testNanWithConstructor() {
+    void testNanWithConstructor() {
         DateTimeStamp dateTimeStamp = new DateTimeStamp((String) null,Double.NaN);
         DateTimeStamp forComparing = new DateTimeStamp(0.0);
         assertNotEquals(dateTimeStamp, forComparing);
     }
 
-    // Maybe every time should be after NaN, but it's NaN so IllegalStateException? Is this a use case in the context of a GC log
```

Comment:
I think these should be left in as well.

Reply:
I could be wrong but I don't think these tests make sense with the change in implementation.